x-pack/test/alerting_api_integration/common/lib/es_test_index_tool.ts

```diff
@@ -4,20 +4,18 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export const ES_TEST_INDEX_NAME = '.kibaka-alerting-test-data';
+export const ES_TEST_INDEX_NAME = '.kibana-alerting-test-data';
 
 export class ESTestIndexTool {
-  private readonly es: any;
-  private readonly retry: any;
-
-  constructor(es: any, retry: any) {
-    this.es = es;
-    this.retry = retry;
-  }
+  constructor(
+    private readonly es: any,
+    private readonly retry: any,
+    private readonly index: string = ES_TEST_INDEX_NAME
+  ) {}
```

Comment:
TIL, you can define right in the constructor!

Reply:
Not often we get to do this, since we often pass an object with props as the constructor arg, and I don't think you can use this trick for that.  But I try to use it wherever I can!